modules/apps/portal-security-audit/test.properties

```diff
@@ -9,5 +9,11 @@
 ## Modules
 ##
 
-    modules.includes.required[relevant]=\
-        apps/portal-security
\ No newline at end of file
+    modules.includes.required.test.batch.class.names.includes[modules-integration-*-jdk8][relevant]=\
+        **/portal-security/**/src/testIntegration/**/security/**/test/*Test.java
```

Comment:
@dellawang Can we make these globs look more like what we have for the Security test suite?
```
    test.batch.class.names.includes[security]=\
        **/oauth2-provider/**/*Test.java,\
        **/portal-security/**/*Test.java,\
        **/portal-security-audit/**/*Test.java,\
        **/portal-security-sso/**/*Test.java,\
        **/portal-security-wedeploy-auth/**/*Test.java,\
        **/saml/**/*Test.java,\
        **/test/unit/**/security/*Test.java
```

Reply:
OK,Thank you!